docs/services/development-services/customers-full-reviews.md

```diff
@@ -100,36 +100,104 @@ description: Real feedback from real clients. Dive into our testimonials to unde
             <div class="section-title">Ready to launch your unique product?</div>
             <div class="section-text">Leave us your name and email and we will get back to you within 2 business days</div>
         </div>
-        <form id="contact-form" class="contact-form" method="post" onsubmit="return validateContactForm(this)">
+        <form class="contact-form">
             <fieldset>
-                <div class="form-section">
-                    <div class="form-element">
-                        <label for="name">
-                            <input id="name" class="cdu-form-control" value="" placeholder="Your Name" name="name" type="text" size="40" maxlength="50">
-                            <p>Name*</p>
-                        </label>
-                    </div>
-                    <div class="form-element">
-                        <label for="email">
-                            <input id="email" class="cdu-form-control" value="" placeholder="Enter Email" name="email" type="email" size="40" maxlength="80">
-                            <p>Email Address*</p>
-                        </label>
-                    </div>
-                </div>
                 <div class="submit-button-container">
-                    <input class="cdu-button" value="Submit" type="submit">
+                    <input class="cdu-button" value="Get in touch" type="button" id="myBtn">
                 </div>
             </fieldset>
         </form>
     </div>
-    <!-- <svg id="rectangle3" class="bg-rectangle"><rect/></svg> -->
-    <!-- <svg id="rectangle4" class="bg-rectangle"><rect/></svg> -->
 </section>
 
+<div id="myModal" class="modal">
+  <div class="modal-content">
+    <div class="close-button">
+        <img class="close" src="/images/close-icon.svg" alt="Close"/>
+    </div>
```

Comment:
Maybe it is better that this icon is fixed when scrolling the content?

Reply:
Added.